core/src/main/java/com/amplifyframework/core/model/AuthStrategy.java

```diff
@@ -75,28 +75,33 @@ public int getPriority() {
     }
 
     /**
-     * Represents the the value of the provider field of the @auth directive.
+     * Represents the value of the provider field of the @auth directive.
      */
     public enum Provider {
         /**
-         * The userPools provider of the @auth rule directive.
+         * The custom provider of the @auth rule directive.
          */
-        USER_POOLS("userPools", 1),
+        CUSTOM("custom", 1),
```

Comment:
This is probably a good call in terms of ranking higher than everything else. Do you know if the other platforms are following suit?

Reply:
Yes, the other platforms are doing the same!   We decided on this order in the design review because they should be ordered by "most specific" to "least specific".    Since you can authorize based on specific attributes of a user in an AWS Lambda function, `CUSTOM` is the most specific.